XamarinCommunityToolkit/Views/GravatarImageSource/UWP/GravatarImageSourceHandler.uwp.cs

```diff
@@ -0,0 +1,25 @@
+using System;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+using Windows.Storage;
+using Windows.UI.Xaml.Media.Imaging;
+using Xamarin.Forms.Platform.UWP;
+
+namespace Xamarin.CommunityToolkit.UI.Views
+{
+	public class GravatarImageSourceHandler : IImageSourceHandler
+	{
+		public async Task<Windows.UI.Xaml.Media.ImageSource> LoadImageAsync(Xamarin.Forms.ImageSource imagesource, CancellationToken cancellationToken = default)
+		{
+			var fileInfo = await GravatarHandlerUtil.Load(imagesource, 1, ApplicationData.Current.LocalCacheFolder.Path).ConfigureAwait(false);
+
+			if (fileInfo?.Exists ?? false)
+			{
```

Comment:
We don't use braces for the one-line statement. Please fix the others, to match our code style.